test/integration/cms/api/templates_test.rb

```diff
@@ -135,6 +135,22 @@ def setup
         assert_equal '/cool', doc.xpath('/page/path').text
       end
 
+      {
+        HTML: '<html><body><p>This is a test</p></body></html>',
+        CDATA: '<![CDATA[<html><body><p>This is a test</p></body></html>]]>'
+      }.each do |name, published|
+        test "show a page with CDATA + #{name} content" do
```

Comment:
nitpick
```suggestion
        test "show a page with #{name} inside the content" do
```

Reply:
I committed this from the Github UI, first time ever!